### ISSUE TITLE
[Item] Unidentifiable Organ

### ISSUE THREAD

**Issue by Byssarod** [dalaranwow/dalaran-wow#4896]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
While player have equipped trinket item Unidentifiable Organ (https://wowgaming.altervista.org/aowow?item=50341), the proc of trinked Invigorated (https://wotlk.evowow.com/?spell=71575#triggered-by) is only happening on incoming hits which player blcoks.
**Expected behaviour**: Tell us what should happen instead
The trinket should proc from only physical melee hits and not fully blocked physical melee hits.
**Steps to reproduce the problem**:

1. Equip Unidentifiable Organ
2. Let NPC melee hits you
3. Watch combat text and buffs

**Include proofs for this behaviour**
http://www.wowhead.com/item=50341/unidentifiable-organ#comments
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
Unidentifiable Organ (https://wowgaming.altervista.org/aowow?item=50341)
Invigorated (https://wotlk.evowow.com/?spell=71575#triggered-by)
You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello @Byssarod, thank you for the report.
Confirmed on ptr.
![wowscrnshot_101517_130327](https://user-images.githubusercontent.com/6797611/31583737-48d9556c-b1a9-11e7-88c3-b077db1ac5bf.jpg)
![wowscrnshot_101517_130428](https://user-images.githubusercontent.com/6797611/31583744-674620ca-b1a9-11e7-9c5b-9f008f729f08.jpg)

**Comment by idontreallyknow** [comment]
would be great if it could be fixed soon..almost useless for druids for awhile now

**Comment by Rushor** [comment]
As you can see in http://web.archive.org/web/20120709041437/http://maintankadin.failsafedesign.com/forum/viewtopic.php?f=21&t=26800 it will now proc on blocks and hits
